packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,12 +101,25 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether the announcement should interrupt any existing announcement,
+  /// or queue after it.
+  ///
+  /// On the web this option uses the aria-live level to set the assertiveness
+  /// of the announcement. On iOS, Android, Windows, Linux, macOS, and Fuchsia
+  /// this option currently has no effect.
+  final Assertiveness assertiveness;
+
   @override
   Map<String, dynamic> getDataMap() {
-    return <String, dynamic>{
+    final Map<String, dynamic> dataMap = <String, dynamic> {
       'message': message,
       'textDirection': textDirection.index,
     };
+
+    if (assertiveness == Assertiveness.assertive) {
```

Comment:
You can use conditional elements to avoid having a separate `if` block:

```dart
  return <String, dynamic> {
    'message': message,
    'textDirection': textDirection.index,
    if (assertiveness != Assertiveness.polite)
      'assertiveness': assertiveness.index,
  };
```

I also recommend using `assertiveness != Assertiveness.polite` instead of `assertiveness == Assertiveness.assertive` because if we add a new enum value, the `==` variant will lead to a bug.